content/src/main/content/jcr_root/_oak_index/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:oak="http://jackrabbit.apache.org/oak/ns/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" jcr:primaryType="nt:unstructured">
+        <cqAuditEvent
```

Comment:
I guess my concern w auto-deploying an index is a pretty intrusive operation. If someone doesn't want to use this feature they still have the cost/risk of managing this index...

I wonder if we make a check that displays on the tool page warning if the index isn't created, and in docs we provide an ensure oak index definition that can be deployed to create it?